src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  if (days > 2 && days < 7) {
+    return days * rentPrice - shortTermDiscount;
```

Comment:
The condition checks for a range between 2 and 7 days but doesn't use constants for these values. Replace the literals with named constants to improve code readability and maintainability.